### DISCUSSION TITLE
difficulty in understanding the code

### DISCUSSION THREAD

**Discussion by n1kh1lmore** [Devansh-47/Sign-Language-To-Text-and-Speech-Conversion#1] [General]
hello Devansh,
This project is so amazing I found it from YT recommendation, its a very nice project I run on my system also, and the way you used mediapipe and other packages is amazing, but I am facing problems in understanding the code and how you used mediapipe so nicely which accurately recognizes, if you are available then please tell me and teach me how does this project run in so deeply or if you can make a tutorial video on youtube It is also good for me as well as everyone

**Comment by DarkerCoder** [comment]
what are the steps of doing this project.. 
Can anyone tell me the perfect path??
